src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -44,4 +44,18 @@ public Navigate toPage(String pageName, String reference) {
 
     return this;
   }
+
+  public Navigate toPage_1(String pageName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)